drake/examples/Valkyrie/+valkyrieParams/Walking.m

```diff
@@ -0,0 +1,45 @@
+classdef Walking < valkyrieParams.Base
+  methods
+    function obj = Walking(r)
+      typecheck(r, 'Valkyrie');
+      obj = obj@valkyrieParams.Base(r);
+      obj.contact_threshold = 0.001;
+      obj.whole_body.w_qdd = zeros(r.getNumVelocities(), 1);
+      if (r.getNumVelocities() ~= r.getNumPositions())
+        error('this code calls findPositionIndices, which is no longer equivalent to findVelocityIndices');
+      end
+
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Hip')) = 1e-6;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Knee')) = 1e-6;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Ankle')) = 1e-6;
+
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Shoulder')) = .0001;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Elbow')) = .0001;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Forearm')) = .0001;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'Wrist')) = .0001;
+
+      obj.whole_body.w_qdd(findPositionIndices(r, 'torso')) = .0001;
+
+      obj.whole_body.w_qdd(r.findPositionIndices('torsoRoll')) = 0.1;
+
+      %This gain was added specifically for Valkyrie.
+      % There must be something else going on as it shouldn't be needed.
+      obj.whole_body.w_qdd(findPositionIndices(r, 'lowerNeckPitch')) = .0001;
```

Comment:
I needed to add this control to the neck to be stable, but I don't see why it was necessary as it wasnt needed for Atlas.